### PR TITLE
[fixed] Dragging inside modal and release outside of modal closes the modal

### LINKED ISSUE #466: Dragging inside modal and release outside of modal closes the modal
### Summary:

Dragging inside modal and release outside of modal closes the modal

### Steps to reproduce:

 1. Drag something inside of a modal
 2. Release the mouse outside the modal
 3. Modal closes due to detecting a "click" outside the modal

### Expected behavior:

Modal should not respond to mouse up that didn't start inside of the overlay

### Link to example of issue:

https://codesandbox.io/s/mgwy6V6E

### Additional notes:

I noticed this while using a draggable interface component inside of the modal. It's natural to sometimes overshoot the modal and drag onto the overlay. This results in an unexpected close of the modal.

### PR DESCRIPTION
**Pull request by tsmirnov**
Fixes #466.

### CODE REVIEW

File: src/components/ModalPortal.js [hunk 1]
```diff
@@ -221,6 +221,10 @@ export default class ModalPortal extends Component {
     this.shouldClose = false;
   }
 
+  handleCountenOnMouseDown = () => {
```
Comment (diasbruno) [inline src/components/ModalPortal.js]: Typo: `handleCountenOnMouseDown` > `handleContentOnMouseDown`.

**Comment by tsmirnov** [comment]
@diasbruno do you know, when this PR might be merged and released?

**Comment by diasbruno** [comment]
@tsmirnov Right now!

**Comment by diasbruno** [comment]
@tsmirnov Thank you!

**Comment by diasbruno** [comment]
Released `v2.2.4`.

**Comment by diasbruno** [comment]
@tsmirnov Please, notify the issues related to this issue.

**Comment by tsmirnov** [comment]
@diasbruno What do you mean?

**Comment by diasbruno** [comment]
Just give a heads up on the issues related to this. :)

**Comment by tsmirnov** [comment]
#131, #383, #466. That is all I found.

**Pull request merged**